### ISSUE TITLE
"Remove Text from HI" Option works incorrectly in "Batch Convert" Mode

### ISSUE THREAD

**Issue by MosDos96** [SubtitleEdit/subtitleedit#2957]
**NOTE: I've tried the latest beta version and it's the same.**

For Example:
In Batch Mode,, If we check "_Remove Text from HI_" and :

- if we check "_Remove Interjections_" and its sub-checkbox "_Only If on separate line_", it works even if the  Interjection is NOT in separate line!
- if we check "_Only If ...._" sub-CheckBoxes, it works even if we don't check their Parent-CheckBoxes!
- if we check "Remove Text from HI" and Select NOTHING in its sub window It does everything like "_Remove Text Between_", "_Remove Interjections_",etc...!

**In summary** >> I think It needs some fix in batch mode

**Comment by niksedk** [comment]
Thx for the info :)

I've tried to fix it in latest beta - could you verify?
https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.6/SubtitleEditBeta.zip

**Comment by MosDos96** [comment]
U R welcome :)

Thx for the new build, I ran some tests, and I think It has been fixed now!

_But there's something I've noticed, in both "Batch Convert Mode" and "Normal Mode":_
If we check "Remove text before a colon" and the line has a **"br/"** tag (AKA: It's split into to sub-lines) and the colon is on 2nd sub-line, that option doesn't work at all, furthermore, if the 1st sub-line has **{\anX}** tag, it'll be removed by mistake.

![se01](https://user-images.githubusercontent.com/31675436/42308187-26dee962-8035-11e8-83f0-cb13dd19566d.png)

**SRT file for test:**
[https://drive.google.com/file/d/1MtvRiEY5D2l1NA6cj13JR0TcI5DsIH--/view?usp=sharing](url)

**Comment by niksedk** [comment]
Thx again - beta updated: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.6/SubtitleEditBeta.zip
